src/Equinor.ProCoSys.IPO.Domain/EntityBase.cs

```diff
@@ -32,8 +41,12 @@ public virtual void SetRowVersion(string value)
             }
         }
 
-        public void RemoveDomainEvent(INotification eventItem) => _domainEvents?.Remove(eventItem);
+        public void RemovePreSaveDomainEvent(INotification eventItem) => _preSaveDomainEvents?.Remove(eventItem);
```

Comment:
Remove, not in use (other than in 1 test)
Also remove RemovePostSaveDomainEvent